### PR TITLE
Updates to GitOps Release Notes for 4-7-GA

### PR DESCRIPTION
**Pull request by Srivaralakshmi**
- **Aligned team**: Dev Tools
- **OCP version for cherry-picking**: `enterprise-4.7` and `enterprise-4.8`
- **JIRA issues**: [RHDEVDOCS-2788](https://issues.redhat.com/browse/RHDEVDOCS-2788)
- **Preview pages**: [Red Hat OpenShift GitOps release notes](https://deploy-preview-31642--osdocs.netlify.app/openshift-enterprise/latest/cicd/gitops/gitops-release-notes.html)
- **Reviewer**: @wtam2018, @reginapizza, @ishitasequeira

### CODE REVIEW

File: cicd/gitops/gitops-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
+//OpenShift Pipelines Release Notes
```
Comment (Preeticp) [inline cicd/gitops/gitops-release-notes.adoc:1]: Remove this

Comment (Preeticp) [inline cicd/gitops/gitops-release-notes.adoc:1]: Remove this

File: cicd/gitops/gitops-release-notes.adoc [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+include::modules/gitops-document-attributes.adoc[]
+[id="gitops-release-notes"]
+= {gitops-title} release notes
+:context: gitops-release-notes
+include::modules/common-attributes.adoc[]
```
Comment (Preeticp) [inline cicd/gitops/gitops-release-notes.adoc:6]: You will need to add an include statement for modules/gitops-document-attributes.adoc too

File: cicd/gitops/gitops-release-notes.adoc [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+toc::[]
+
+{gitops-title} is a cloud-native CI/CD experience based on the Tekton project which helps you automate the following tasks:
```
Comment (Preeticp) [inline cicd/gitops/gitops-release-notes.adoc:10]: Please use https://docs.openshift.com/container-platform/4.7/cicd/gitops/understanding-openshift-gitops.html#key-features_understanding-openshift-gitops and https://docs.openshift.com/container-platform/4.7/cicd/gitops/understanding-openshift-gitops.html#about-redhat-openshift-gitops_understanding-openshift-gitops to add a couple of introductory sentences here.

File: modules/gitops-release-notes-1-1.adoc [hunk 4]
```diff
@@ -0,0 +1,54 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-1_{context}"]
+= Release notes for {gitops-title} 1.1
+
+[id="gitops-features-1-1_{context}"]
+== GitOps features
```
Comment (Preeticp) [inline modules/gitops-release-notes-1-1.adoc:9]: GitOps Technology Preview features

File: modules/gitops-release-notes-1-1.adoc [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+.GitOps features
```
Comment (Preeticp) [inline modules/gitops-release-notes-1-1.adoc:11]: GitOps technology preview features

File: modules/gitops-release-notes-1-1.adoc [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+[cols="1,1",options="header"]
+|===
+| {gitops-title} Product | Release
+| Argo CD
+| General availability (GA)
+| Argo CD ApplicationSet
+| Tech Preview (TP)
```
Comment (Preeticp) [inline modules/gitops-release-notes-1-1.adoc:18]: Technology preview in all instances.

Comment (Preeticp) [reply modules/gitops-release-notes-1-1.adoc:18]: We can add a key like the core OCP RN does: In the table below, features are marked with the following statuses:

TP: Technology Preview

GA: General Availability

File: modules/gitops-release-notes-1-1.adoc [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+| {gitops-title} Application Manager (kam)
+| Tech Preview (TP)
+| {gitops-title} Service
+| Tech Preview (TP)
+|===
+
+
+[id="new-features-1-1_{context}"]
+== New features
+{gitops-title} 1.1 is now available on {product-title} 4.7. {gitops-title} 1.1 is updated to support:
```
Comment (Preeticp) [inline modules/gitops-release-notes-1-1.adoc:28]: Remove: {gitops-title} 1.1 is updated to support: 
Also remove bullet points on lines 30-33

File: modules/gitops-release-notes-1-1.adoc [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+*
+*
+*
+*
+
+In addition to the fixes and stability improvements, the following sections highlight what is new in {gitops-title} 1.1.
+
```
Comment (Preeticp) [inline modules/gitops-release-notes-1-1.adoc:36]: Add the new features here.

File: cicd/gitops/gitops-release-notes.adoc (hunk 1 above)
Comment (Srivaralakshmi) [reply cicd/gitops/gitops-release-notes.adoc:1]: Done

File: cicd/gitops/gitops-release-notes.adoc (hunk 3 above)
Comment (Srivaralakshmi) [reply cicd/gitops/gitops-release-notes.adoc:10]: Done.

File: modules/gitops-release-notes-1-1.adoc (hunk 7 above)
Comment (Srivaralakshmi) [reply modules/gitops-release-notes-1-1.adoc:28]: Done

File: modules/gitops-release-notes-1-1.adoc (hunk 4 above)
Comment (Srivaralakshmi) [reply modules/gitops-release-notes-1-1.adoc:9]: Done

File: modules/gitops-release-notes-1-1.adoc (hunk 5 above)
Comment (Srivaralakshmi) [reply modules/gitops-release-notes-1-1.adoc:11]: Done

File: modules/gitops-release-notes-1-1.adoc (hunk 6 above)
Comment (Srivaralakshmi) [reply modules/gitops-release-notes-1-1.adoc:18]: Done

File: modules/gitops-release-notes-1-1.adoc (hunk 8 above)
Comment (Srivaralakshmi) [reply modules/gitops-release-notes-1-1.adoc:36]: Done

**Comment by Srivaralakshmi** [comment]
@Preeticp @wtam2018, @reginapizza, @ishitasequeira PTAL and provide your comments.

File: modules/gitops-release-notes-1-1.adoc [hunk 9]
```diff
@@ -0,0 +1,129 @@
...
+     uid: f5226fbc-883d-47db-8b53-b5e363f007af
+     creationTimestamp: '2021-04-16T19:24:08Z'
+     managedFields:
+    ...
+     namespace: openshift-gitops
+     labels:
+       app.kubernetes.io/managed-by: argocd-cluster
+       app.kubernetes.io/name: argocd-cm
+       app.kubernetes.io/part-of: argocd
+       data:                                            "" <1>
```
Comment (wtam2018) [inline modules/gitops-release-notes-1-1.adoc:92]: `data:` should line up with `metadata:`.  Everything else looks good.

File: modules/gitops-release-notes-1-1.adoc [hunk 10]
```diff
@@ -0,0 +1,129 @@
...
+== New features
+{gitops-title} 1.1 is now available on {product-title} 4.7. These are the new features in {gitops-title} 1.1:
+
+* The `ApplicationSet` feature is now added (Technology Preview). The `ApplicationSet` feature enables both automation and greater flexibility when managing Argo CD applications across a large number of clusters and within monorepos. It also makes self-service usage possible on multitenant Kubernetes clusters.
+* Argo CD is now integrated with cluster logging stack and with the {product-title} Monitoring and Alerting features.
+* Argo CD auth is now integrated with {product-title}.
+* Argo CD applications controller now supports horizontal scaling.
+* Argo CD Redis servers now support high availability (HA).
+
+In addition to the new features, the following sections highlight the fixes and stability improvements in {gitops-title} 1.1.:
```
Comment (Preeticp) [inline modules/gitops-release-notes-1-1.adoc:43]: Either keep the ./:

File: modules/gitops-release-notes-1-1.adoc [hunk 11]
```diff
@@ -0,0 +1,129 @@
...
+
+[id="fixed-issues-1-1_{context}"]
+== Fixed issues
+The following issues been resolved in the current release:
```
Comment (Preeticp) [inline modules/gitops-release-notes-1-1.adoc:47]: been/were

File: modules/gitops-release-notes-1-1.adoc [hunk 12]
```diff
@@ -0,0 +1,129 @@
...
+* Previously, the {gitops-title} backend failed to find the valid GitLab repository. This was because the {gitops-title} backend checked for `main` as the branch reference, instead of `master` in the GitLab repository. This issue is fixed now. link:https://issues.redhat.com/browse/GITOPS-768[GITOPS-768]
+* The *Environments* page in the {product-title} web console from the *Developer* perspective now shows the list of applications and the number of environments along with an Argo CD link that directs you to the Argo CD *Applications* page that lists all the applications. The Argo CD *Applications* page has *LABELS* (for example, `app.kubernetes.io/name=appName`) that helps you filter only the applications of your choice to be shown in that view. link:https://issues.redhat.com/browse/GITOPS-544[GITOPS-544]
```
Comment (Preeticp) [inline modules/gitops-release-notes-1-1.adoc:52]: Split the first sentence it is too huge: The *Environments* page in the {product-title} web console from the *Developer* perspective now shows a list of applications and the number of environments. It also displays an Argo CD link that directs you to the Argo CD *Applications* page that lists all the applications.
helps/help
remove `to be shown in that view`

File: modules/gitops-release-notes-1-1.adoc [hunk 13]
```diff
@@ -0,0 +1,129 @@
...
+
+
+[id="known-issues-1-1_{context}"]
+== Known issues
+These are the known issues in {gitops-title} 1.1:
+
+* {gitops-title} does not support Helm v2 and ksonnet.
+* Red Hat SSO (RH SSO) Operator is not supported in disconnected clusters. As a result, the {gitops-title} Operator and RH SSO integration is not supported in disconnected clusters.
```
Comment (Preeticp) [inline modules/gitops-release-notes-1-1.adoc:60]: The Red Hat SSO

File: modules/gitops-release-notes-1-1.adoc [hunk 14]
```diff
@@ -0,0 +1,129 @@
...
+        url: >-  https://argocd-cluster-server-openshift-gitops.apps.dev-svc-4.7-041614.devcluster.openshift.com   "" <2>
+        help.chatText: ''
+        kustomize.buildOptions: ''
+        resource.inclusions: ''
+        repository.credentials: ''
+        users.anonymous.enabled: 'false'
+        configManagementPlugins: ''
+        application.instanceLabelKey: ''
+----
+<1> Restore only the _data_ section of the content in the `argocd-cm.yml` config map file manually.
```
Comment (Preeticp) [inline modules/gitops-release-notes-1-1.adoc:118]: `data`

File: modules/gitops-release-notes-1-1.adoc [hunk 15]
```diff
@@ -0,0 +1,129 @@
...
+<2> Replace the URL value in the config map entry with the new instance name `openshift-gitops`.
+
+. Delete the default `argocd-cluster` instance.
+. Edit the new `argocd-cm.yml` config map file to restore the entire _data_ section manually.
```
Comment (Preeticp) [inline modules/gitops-release-notes-1-1.adoc:122]: `data`

File: modules/gitops-release-notes-1-1.adoc [hunk 16]
```diff
@@ -0,0 +1,129 @@
...
+. Replace the URL value in the config map entry with the new instance name `openshift-gitops`. For example, in the preceding example, replace the URL value with the following URL value:
++
+[source,yaml]
+----
+        url: >-  https://openshift-gitops-server-openshift-gitops.apps.dev-svc-4.7-041614.devcluster.openshift.com
```
Comment (Preeticp) [inline modules/gitops-release-notes-1-1.adoc:127]: We can remove the extra space before the `url` field here.
Also, I am not sure why this is highlighted in red in the preview, here as well as in the YAML snippet above.

File: modules/gitops-release-notes-1-1.adoc [hunk 17]
```diff
@@ -0,0 +1,129 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-1_{context}"]
+= Release notes for {gitops-title} 1.1
+
+[id="gitops-technology-preview-features-1-1_{context}"]
+== GitOps Technology Preview features
```
Comment (Preeticp) [inline modules/gitops-release-notes-1-1.adoc:9]: Drop the GitOps, does not seem necessary.

**Comment by Preeticp** [comment]
Some minor nits @Srivaralakshmi otherwise looks good.

File: modules/gitops-release-notes-1-1.adoc (hunk 9 above)
Comment (Srivaralakshmi) [reply modules/gitops-release-notes-1-1.adoc:92]: Done

File: modules/gitops-release-notes-1-1.adoc (hunk 10 above)
Comment (Srivaralakshmi) [reply modules/gitops-release-notes-1-1.adoc:43]: Done

File: modules/gitops-release-notes-1-1.adoc (hunk 11 above)
Comment (Srivaralakshmi) [reply modules/gitops-release-notes-1-1.adoc:47]: Done

File: modules/gitops-release-notes-1-1.adoc (hunk 12 above)
Comment (Srivaralakshmi) [reply modules/gitops-release-notes-1-1.adoc:52]: Done

File: modules/gitops-release-notes-1-1.adoc (hunk 13 above)
Comment (Srivaralakshmi) [reply modules/gitops-release-notes-1-1.adoc:60]: Done

File: modules/gitops-release-notes-1-1.adoc (hunk 14 above)
Comment (Srivaralakshmi) [reply modules/gitops-release-notes-1-1.adoc:118]: Done

File: modules/gitops-release-notes-1-1.adoc (hunk 15 above)
Comment (Srivaralakshmi) [reply modules/gitops-release-notes-1-1.adoc:122]: Done

File: modules/gitops-release-notes-1-1.adoc (hunk 16 above)
Comment (Srivaralakshmi) [reply modules/gitops-release-notes-1-1.adoc:127]: Done

File: modules/gitops-release-notes-1-1.adoc (hunk 17 above)
Comment (Srivaralakshmi) [reply modules/gitops-release-notes-1-1.adoc:9]: Done

**Comment by Srivaralakshmi** [comment]
@wtam2018 and @Preeticp Incorporated all the comments. PTAL and kindly provide LGTM if the content looks good. Thanks!

File: modules/gitops-release-notes-1-1.adoc (hunk 9 above)
Comment (Srivaralakshmi) [reply modules/gitops-release-notes-1-1.adoc]: Done

**Review by wtam2018 — APPROVED** [review]
LGTM 
thanks @Srivaralakshmi

**Review by Preeticp — COMMENTED** [review]
@Srivaralakshmi a couple of minor nits and we are good to merge. Looks great otherwise. Thank you!!

File: modules/gitops-release-notes-1-1.adoc [hunk 18]
```diff
@@ -0,0 +1,129 @@
...
+* Previously, the {gitops-title} backend failed to find the valid GitLab repository. This was because the {gitops-title} backend checked for `main` as the branch reference, instead of `master` in the GitLab repository. This issue is fixed now. link:https://issues.redhat.com/browse/GITOPS-768[GITOPS-768]
+* The _Environments_ page in the {product-title} web console from the _Developer_ perspective now shows the list of applications and the number of environments. This page also displays an Argo CD link that directs you to the Argo CD _Applications_ page that lists all the applications. The Argo CD *Applications* page has *LABELS* (for example, `app.kubernetes.io/name=appName`) that help you filter only the applications of your choice. link:https://issues.redhat.com/browse/GITOPS-544[GITOPS-544]
```
[the remark this answers is not in the document]
Comment (Srivaralakshmi) [reply modules/gitops-release-notes-1-1.adoc:52]: Yes, Thanks for the clarification. Done.

**Pull request merged**